crates/core/src/task_center.rs

```diff
@@ -642,17 +714,17 @@ impl TaskCenter {
                     {
                         // The task failed to spawn other tasks because the system
                         // is already shutting down, we ignore those errors.
-                        tracing::debug!(kind = ?kind, name = ?task.name, "[Shutdown] Task {} stopped due to shutdown", task_id);
+                        tracing::debug!(kind = ?task.kind(), name = ?task.name(), "[Shutdown] Task {} stopped due to shutdown", task_id);
```

Comment:
A possible future improvement is to instrument the wrapper and include kind, name, and other task-level attributes in the Span itself.